tonic-build/src/prost.rs

```diff
@@ -368,6 +370,14 @@ impl Builder {
         self
     }
 
+    /// Enable or disable emitting the rerun-if-changed cargo instruction
+    /// 
+    /// If set, emits rerun-if-changed instructions so that Cargo understands when to rerun the build script.
```

Comment:
Is this the only place I need to document this option?

Reply:
Yeah I think this is fine, I would also add info about the default setting that if you're in a build script this will be enabled and outside of a build script it won't